src/utils/ui.js

```diff
@@ -0,0 +1,9 @@
+/* Checks to see if the root element is available and
+ * if the element is visible. We check the width of the element
+ * because it is more reliable than either checking a focus state or
+ * the visibleState or hidden property.
+ */
+export function isVisible() {
```

Comment:
does `isVisible` feels vague, maybe something like `isAppVisible` is a bit clearer?